app/components/shared/widget-inputs/WTextAreaInput.vue.ts

```diff
@@ -15,5 +16,9 @@ export default class WTextAreaInput extends WInput<string, IWTextMetadata> {
   @Prop({ default: () => ({}) })
   metadata: IWTextMetadata;
 
-
+  getValidations() {
+    return {
+      ...super.getValidations(),
+    };
+  }
```

Comment:
Hmm, do we really need that here?

Reply:
true :) just copy pasted. removing now

Reply:
WTextAreaInput extends WInput that already has `getValidations` method